02Lesson/01Project/index.html

```diff
@@ -1,9 +1,37 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
+  <head>
+    <title>J&B Coffee Co.</title>
+  </head>
+  <body>
+    <img src="http://restaurantguidelubbock.com/wp-content/uploads/2012/07/01-J-b-coffee-co.jpg"/>
+    <h1>J&B Coffee</h1>
+    <br>
+    <h2>ABOUT US</h2>
+    <p>We love coffee and we love the Lubbock community.
+      Whether you need a study or meeting location,
+      or even a place to throw your next shower -
+      we have the space you need.</p>
+      <br>
```

Comment:
Indentation: I would always put block elements in their own line:
```html
<p>
  We love ....
</p>
```
Then you see immediately that the `br` tag is indented too much.